compile.sh

```diff
@@ -1,20 +1,23 @@
-#!/bin/bash
+#!/usr/bin/env sh
 
 # Compile jq to WebAssembly
 
 # Need Emscripten installed
-[[ ! $(command -v emcc --version) ]] && \
-  echo "You need Emscripten to compile jq to WebAssembly. See the setup instructions in the README file." && \
-  exit
+if ! command -v emcc > /dev/null 2>&1; then
+  printf '%s\n' \
+    'You need Emscripten to compile jq to WebAssembly.' \
+    'See the setup instructions in the README file.'
+  exit 2
+fi >&2
```

Comment:
Can you undo lines 6-11? Not a fan of printf instead of echo 🙂

Reply:
The reason to not use `echo` is because this command behaves differently in different shells.
There is a good detailed explanation here https://www.in-ulm.de/~mascheck/various/echo+printf/